Casks/puppet-bolt.rb

```diff
@@ -3,27 +3,27 @@
   when '10.11'
     os_ver = '10.11'
     version '2.0.0'
-    sha256 'a5956c7d075a6219f04cda8890732bb8e31b4e10ad0096e18046531b43cfdd7d'
+    sha256 'a0fcb0a8d100ff9294ef7d6d9c00c569c97fa3c25bc857d08b4c6b53aaf2e50d'
```

Comment:
This is kind of a bad idea. Changing checksums of installed images can be really annoying for users to fix.